code/datums/mutations/speech.dm

```diff
@@ -33,6 +38,148 @@
+	if(!message)
+		return
+	// Split for swapping purposes
+	message = " [message] "
+
+	// Splitting up each word in the text to manually apply our intended changes
+	var/list/message_words = splittext(message, " ")
+	// What we use in the end
+	var/list/edited_message_words
+
+	for(var/editing_word in message_words)
+		if(editing_word == " " || editing_word == "" )
+			continue
+		// Used to replace the original later
+		var/og_word = editing_word
+		// Iterating through each replaceable-string in the .json
+		var/list/static/super_wacky_words = strings("heckacious.json", "heckacious")
+
+		// If the word doesn't get replaced we might do something with it later
+		var/word_edited
+		for(var/key in super_wacky_words)
+			var/value = super_wacky_words[key]
+			// If list, pick one value from said list
+			if(islist(value))
+				value = pick(value)
+			editing_word = replacetextEx(editing_word, "[uppertext(key)]", "[uppertext(value)]")
+			editing_word = replacetextEx(editing_word, "[capitalize(key)]", "[capitalize(value)]")
+			editing_word = replacetextEx(editing_word, "[key]", "[value]")
+			// Enable if we actually found something to change
+			if(editing_word != og_word)
+				word_edited = TRUE
+
+		// Random caps
+		if(prob(10))
+			editing_word = uppertext(editing_word)
+		// some times....... we add DOTS...
+		if(prob(10))
+			for(var/dotnum in 1 to rand(2, 8))
+				editing_word += "."
+
+		// If no replacement we do it manually
+		if(!word_edited)
+			if(prob(65))
+				editing_word = replacetext(editing_word, pick(VOWELS), pick(VOWELS))
+			// Many more consonants, double it!
+			for(var/i in 1 to rand(1, 2))
+				editing_word = replacetext(editing_word, pick(CONSONANTS), pick(CONSONANTS))
+			// rarely, lettter is DOUBBLED...
+			var/patchword = ""
+			for(var/letter in 1 to length(editing_word))
+				if(prob(92))
+					patchword += editing_word[letter]
+					continue
+				patchword += replacetext(editing_word[letter], "", editing_word[letter] + editing_word[letter])
+			editing_word = patchword
+
+		// Some words are randomly recolored and resized so they get a few of these
+		// span combinations will not make multiple colors yay
+		var/list/static/span_combo_list = list("big", "small", "green", "red", "blue")
+		if(prob(15))
+			editing_word = "<span class='[pick(span_combo_list)]'>" + editing_word + "</span>"
+
+		LAZYADD(edited_message_words, editing_word)
+
+	var/edited_message = jointext(edited_message_words, " ")
+
+	message = trim(edited_message)
+
+	speech_args[SPEECH_MESSAGE] = message
+
+/datum/mutation/human/trichromatic
+	name = "Trichromatic Larynx"
+	desc = "A strange mutation originating from Clown Planet which alters the color of the patient's vocal chords."
+	quality = MINOR_NEGATIVE
+	text_gain_indication = "<span class='red'>You.</span> <span class='blue'>Feel.</span> <span class='green'>Weird.</span>"
+	text_lose_indication = "<span class='notice'>Your colors feel normal again.</span>"
```

Comment:
add/use the span class defines

Reply:
https://github.com/tgstation/tgstation/blob/3ad7731897ac9e249562bd8ca414c04287af1ac4/code/__DEFINES/span.dm

You're looking for the color spans and ``span_notice``.